nixos/lib/test-driver/test-driver.py

```diff
@@ -710,6 +710,13 @@ def unblock(self):
         """
         self.send_monitor_command("set_link virtio-net-pci.1 on")
 
+    def copy_file_from_host(self, from_path, to_path):
+        """Copy a file to the machine.
+        """
+        with open(from_path, "r") as f:
```

Comment:
should this not be read as bytes?

Reply:
yeah i thought so to. but would that work with `echo` ?

Reply:
you could use base64 instead

Reply:
done

Reply:
@FRidh @symphorien can this be considered as resolved?